rotkehlchen/assets/converters.py

```diff
@@ -879,7 +889,10 @@ def asset_from_bitstamp(bitstamp_name: str) -> AssetWithOracles:
     if not isinstance(bitstamp_name, str):
         raise DeserializationError(f'Got non-string type {type(bitstamp_name)} for bitstamp asset')
 
-    name = BITSTAMP_TO_WORLD.get(bitstamp_name, bitstamp_name)
+    # (Yabir): Do un upper here since in the tests I've seen that we pass as asset lowercase
+    # symbols and in the mapping we use uppercase for the search. Before it was not an issue
+    # since there were no multiple asset with symbol for example USDC.
+    name = BITSTAMP_TO_WORLD.get(bitstamp_name.upper(), bitstamp_name)
```

Comment:
I don't understand. Can you explain in more detail?

I think bitstamp has mixed case and we used to depend on identifiers? In the DB identifier comparison was at least up to last release case insensitive

Reply:
Here the mapping is from id to upper case symbol and in bitstamp I saw that sometimes the symbols are lowercase so it was failing to find them because it was lookign for `mapping['usdc']` and this has no value but `mapping['USDC']` does

Reply:
before it was no problem because either the symbols with lowercase ended in the database where lower or upper doesn't mapper or is always upper case and is in our tests where we use it lower case

Reply:
Then please also change the comment

```
# (Yabir): Do un upper here since in the tests I've seen that we pass as asset lowercase
    # symbols and in the mapping we use uppercase for the search. Before it was not an issue
    # since there were no multiple asset with symbol for example USDC.
```

to 

```
# bitstamp assets are read as lowercase from the exchange
```